.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: java
```

Comment:
Because the python script wraps a Java library. So you need both Python and Java in order to run html5validator.